src/main/java/frc/robot/commands/DefaultDrive.java

```diff
@@ -31,30 +30,27 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-     this.m_drive.tankDrive2(Constants.MAX_SPEED * (this.m_robotContainer.getStick().getThrottle()), -Constants.MAX_SPEED * (this.m_robotContainer.getStick().getY()));
+    this.m_drive.tankDrive2(
+        Constants.MAX_SPEED * (this.m_robotContainer.getStick().getThrottle()),
+        -Constants.MAX_SPEED * (this.m_robotContainer.getStick().getY()));
   }
-  //test function for autonomous, not used yet
-  public void autoDrive(boolean stop) 
-  {
+  // test function for autonomous, not used yet
+  public void autoDrive(boolean stop) {
     if (!stop) {
       this.m_drive.tankDrive2(0, 0);
```

Comment:
In the rewrite lets try to call subsystem commands directly rather than through an intermediate function call.

Reply:
most of these changes were done by the formatter, not me

Reply:
@sviridovt this pr was basically to setup ci / basic build things and run basic formatting